utils/events/events.go

```diff
@@ -0,0 +1,18 @@
+package events
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/gogo/protobuf/proto"
+
+	"github.com/axelarnetwork/utils/funcs"
+)
+
+// Emit provides a shorthand to emit an event through the context's event manager
+func Emit(ctx sdk.Context, evs ...proto.Message) {
+	// the tests in this package ensure this will never panic
+	if len(evs) == 1 {
```

Comment:
it is yes. It prevents the sdk from allocating another 1-element array on the heap. Not really groundbreaking in the grand scheme of things but since the function is literally just a wrapper to shorten the call line I might as well do it